string SensorName
string Instruct
float64[] Paramete

---

string Result